selfdrive/car/chrysler/values.py

```diff
@@ -129,12 +129,13 @@ class ChryslerCarInfo(CarInfo):
   }],
 }
 
-FW_VERSIONS = {
```

Comment:
If we just ignore ecus with empty version lists, this can cause other cars to mis-fingerprint as we never invalidate this platform since we just `continue` for all two of its ecus.